### PR TITLE
2.1.4 Release Notes Draft

### PR DESCRIPTION
**Pull request by Colin1841**
## Jira Ticket

Release board: https://d2iq.atlassian.net/jira/dashboards/10022 

https://d2iq.atlassian.net/browse/D2IQ-93206

This is currently in the Konvoy repo, but applies to both Konvoy & Kommander for DKP. After edits and approval, we'll duplicate the file over to the Kommander repo.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4635.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Comment by cmastr** [comment]
Don't we need one for Kommander too? Also, this file is all Kommander words, but it's in the Konvoy repo

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.4 Release Notes
+title: Kommander 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Kommander 2.1.4
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.3 was released on October, nn 2022.**
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:10]:
```suggestion
**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October, nn 2022.**
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.4 Release Notes
+title: Kommander 2.1.4 Release Notes
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:4]: Konvoy

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+menuWeight: 50
+excerpt: View release-specific information for Kommander 2.1.4
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.3 was released on October, nn 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:14]: This whole file says Kommander, not Konvoy

Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:14]: This whole file says Kommander, not Konvoy

**Review by cmastr — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### No apparent upgrade path for 1.X vSphere clusters to 2.X
+
+There is a way to migrate existing vsphere clusters from 1.8 to 2.1. When attempting, this ends in failure due to providerID issues with the migration.
+
+<Need to validate if we are still keeping the following issues from 2.1.3:>
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:74]: Validate issues below - keep or remove these for 2.1.4?

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+## Fixes and Improvements
+
+<Add or remove>
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:25]: Add items or remove section, as applicable

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.5
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### No apparent upgrade path for 1.X vSphere clusters to 2.X
+
+There is a way to migrate existing vsphere clusters from 1.8 to 2.1. When attempting, this ends in failure due to providerID issues with the migration.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:72]:
```suggestion
**D2IQ-90769**
There is a way to migrate existing vsphere clusters from 1.8 to 2.1. When attempting, this ends in failure due to providerID issues with the migration.
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.5
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### No apparent upgrade path for 1.X vSphere clusters to 2.X
+
+**D2IQ-90769**
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:72]:
```suggestion
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+There is a way to migrate existing vsphere clusters from 1.8 to 2.1. When attempting, this ends in failure due to providerID issues with the migration.
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:73]:
```suggestion
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md (hunk 4 above)
Comment (cneth) [reply pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:75]: Keep them, there's no fix for this until 2.2.1, as mentioned in the body of the workaround.

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+## Fixes and Improvements
+
+<Add or remove>
+
+## Component updates
+
+The following services and service components are upgraded to the listed version:
+
+- centralized-grafana: 18.1.3
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:31]: Do these all need to be updated? @bcmendoza can help.

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md (hunk 4 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:72]: these are all for Kommander though. So I'd take them out of these release notes, but add them to the Kommander release notes for 2.1.4.

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md (hunk 9 above)
Comment (gracedo) [reply pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:31]: Newly generated table for this release, please copy and paste the table into this section: https://github.com/mesosphere/dkp-infra-tools/blob/6c64a8dda91cd9865e8084e8c9eab802e7b3cf78/automation/cmd/report/hack/v2.1.4-rc.2.md?plain=1

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+- nvidia: 0.4.3
+- project-grafana-logging: 6.16.14
+- project-grafana-loki: 0.33.1
+- project-logging: 1.0.0
+- prometheus-adapter: 2.11.1
+- reloader: 0.0.99
+- thanos: 0.4.5
+- traefik: 10.3.0
+- traefik-forward-auth: 0.3.2
+- velero: 3.1.5
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:64]:
```suggestion
| Common Application Name | APP ID | Version | Component Versions |
|-------------------------|--------|---------|--------------------|
| Centralized Grafana | centralized-grafana | 18.1.3 | - chart: 18.1.3<br>- prometheus-operator: 0.50.0 |
| Centralized Kubecost | centralized-kubecost | 0.20.0 | - chart: 0.20.0<br>- kubecost: 1.88.0 |
| Cert Manager | cert-manager | 0.2.7 | - chart: 0.2.7<br>- cert-manager: 1.0.3 |
| Dex | dex | 2.9.10 | - chart: 2.9.10<br>- dex: 2.22.0 |
| Dex K8s Authenticator | dex-k8s-authenticator | 1.2.9 | - chart: 1.2.9<br>- dex-k8s-authenticator: 1.2.2 |
| External DNS | external-dns | 2.20.5 | - chart: 2.20.5<br>- external-dns: 0.7.0 |
| Fluent Bit | fluent-bit | 0.16.2 | - chart: 0.16.2<br>- fluent-bit: 1.8.3 |
| Gatekeeper | gatekeeper | 0.6.9 | - chart: 0.6.9<br>- gatekeeper: 3.4.0-rc.1 |
| Gitea | gitea | 4.1.1 | - chart: 4.1.1<br>- gitea: 1.15.3 |
| Grafana Logging | grafana-logging | 6.16.14 | - chart: 6.16.14<br>- grafana: 8.2.1 |
| Grafana Loki | grafana-loki | 0.33.1 | - chart: 0.33.1<br>- loki: 2.2.1 |
| Istio | istio | 1.9.1 | - chart: 1.9.1<br>- istio: 1.9.1 |
| Jaeger | jaeger | 2.21.0 | - chart: 2.21.0<br>- jaeger: 1.22.0 |
| Karma | karma | 2.0.0 | - chart: 2.0.0<br>- karma: 0.70 |
| Kiali | kiali | 1.29.1 | - chart: 1.29.1<br>- kiali: 1.29.1 |
| Knative | knative | 0.3.9 | - chart: 0.3.9<br>- knative: 0.22.3 |
| Kube OIDC Proxy | kube-oidc-proxy | 0.2.5 | - chart: 0.2.5<br>- kube-oidc-proxy: 0.2.0 |
| Kube Prometheus Stack | kube-prometheus-stack | 18.1.3 | - chart: 18.1.3<br>- prometheus-operator: 0.50.0<br>- grafana: 8.2.1<br>- prometheus: 2.28.1<br>- prometheus-alertmanager: 0.22.2 |
| Kubecost | kubecost | 0.20.0 | - chart: 0.20.0<br>- kubecost: 1.88.0 |
| Kubefed | kubefed | 0.9.0 | - chart: 0.9.0<br>- kubefed: 0.9.0 |
| Kubernetes Dashboard | kubernetes-dashboard | 5.0.2 | - chart: 5.0.2<br>- kubernetes-dashboard: 2.3.1 |
| Kubetunnel | kubetunnel | 0.0.8 | - chart: 0.0.8<br>- kubetunnel: 0.0.8 |
| Logging Operator | logging-operator | 3.15.0 | - chart: 3.15.0<br>- logging-operator: 3.15.0 |
| Metallb | metallb | 0.12.2 | - chart: 0.12.2<br>- metallb: 0.8.1 |
| MinIO Operator | minio-operator | 4.1.7 | - chart: 4.1.7<br>- minio-operator: 4.1.3 |
| NFS Server Provisioner | nfs-server-provisioner | 0.6.0 | - chart: 0.6.0<br>- nfs-server-provisioner: 2.3.0 |
| Nvidia | nvidia | 0.4.3 | - chart: 0.4.3<br>- nvidia-device-plugin: 0.2.0 |
| Grafana (project) | project-grafana-logging | 6.16.14 | - chart: 6.16.14<br>- grafana: 8.2.1 |
| Grafana Loki (project) | project-grafana-loki | 0.33.1 | - chart: 0.33.1<br>- loki: 2.2.1 |
| Prometheus Adapter | prometheus-adapter | 2.11.1 | - chart: 2.11.1<br>- prometheus-adapter: 0.8.3 |
| Reloader | reloader | 0.0.99 | - chart: 0.0.99<br>- reloader: 0.0.99 |
| Thanos | thanos | 0.4.5 | - chart: 0.4.5<br>- thanos: 0.17.1 |
| Traefik | traefik | 10.3.0 | - chart: 10.3.0<br>- traefik: 2.5.0 |
| Traefik ForwardAuth | traefik-forward-auth | 0.3.2 | - chart: 0.3.2<br>- traefik-forward-auth: 3.0.2 |
| Velero | velero | 3.1.5 | - chart: 3.1.5<br>- velero: 1.5.2 |
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+| Traefik ForwardAuth | traefik-forward-auth | 0.3.2 | - chart: 0.3.2<br>- traefik-forward-auth: 3.0.2 |
+| Velero | velero | 3.1.5 | - chart: 3.1.5<br>- velero: 1.5.2 |
+
+## Known Issues
+
+The following items are known issues with this release.
+
+
+
+<Need to validate if we are still keeping the following issues from 2.1.3:>
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:75]:
```suggestion
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+| Reloader | reloader | 0.0.99 | - chart: 0.0.99<br>- reloader: 0.0.99 |
+| Thanos | thanos | 0.4.5 | - chart: 0.4.5<br>- thanos: 0.17.1 |
+| Traefik | traefik | 10.3.0 | - chart: 10.3.0<br>- traefik: 2.5.0 |
+| Traefik ForwardAuth | traefik-forward-auth | 0.3.2 | - chart: 0.3.2<br>- traefik-forward-auth: 3.0.2 |
+| Velero | velero | 3.1.5 | - chart: 3.1.5<br>- velero: 1.5.2 |
+
+## Known Issues
+
+The following items are known issues with this release.
+
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:72]:
```suggestion
```

**Review by msdolbey — APPROVED** [review]
👍 
- Yeah, just change that ol' date `nn` and it's a done deal as far as I can tell.
- That new table looks really nice and fancy.
- I don't know if these actually address the vSphere fixes that are here, but I mean, it's probably in there I just didn't realize it.

alright alright alright

File: pages/dkp/kommander/2.1/release-notes/2.1.4/index.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+layout: layout.pug
+navigationTitle: Kommander 2.1.4 Release Notes
+title: Kommander 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Kommander 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Kommander&reg; version 2.1.4 was released on October, nn 2022.**
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/release-notes/2.1.4/index.md:11]:
```suggestion
**D2iQ&reg; Kommander&reg; version 2.1.4 was released on October, 18 2022.**
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+layout: layout.pug
+navigationTitle: Konvoy 2.1.4 Release Notes
+title: Konvoy 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October dd, 2022.**
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:11]:
```suggestion
**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October 18, 2022.**
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.1/release-notes/2.1.4/index.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+layout: layout.pug
+navigationTitle: Kommander 2.1.4 Release Notes
+title: Kommander 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Kommander 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Kommander&reg; version 2.1.4 was released on October, 18 2022.**
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/release-notes/2.1.4/index.md:11]:
```suggestion
**D2iQ&reg; Kommander&reg; version 2.1.4 was released on October, 19 2022.**
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+layout: layout.pug
+navigationTitle: Konvoy 2.1.4 Release Notes
+title: Konvoy 2.1.4 Release Notes
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 2.1.4
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October 18, 2022.**
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md:11]:
```suggestion
**D2iQ&reg; Konvoy&reg; version 2.1.4 was released on October 19, 2022.**
```

**Pull request merged**